src/main/java/com/artipie/asto/Key.java

```diff
@@ -18,6 +19,10 @@
  * Storage key.
  *
  * @since 0.6
+ * @todo #341:30min Add others key operations (exclusion by index, insertion,..).
+ *  We should exclude a part from a storage key by index. We should also insert
+ *  a part to a key by specifying the position (Like we do with {@code List.add(Obj, index)}).
+ *  We also want to get parts of a storage key as list or stream.
```

Comment:
@olenagerasimova I'm not sure if it's the part of origin ticket, isn't it? @baudoliver7 if not, please create a new ticket and remove this `todo`

Reply:
@olenagerasimova just to say we are still waiting for your answer before completing this ticket :)

Reply:
@baudoliver7 @g4s8 I think this is part of the ticket, I've provided corresponding examples in the description

Reply:
@olenagerasimova Thanks :)